### ISSUE TITLE
Cross entropy always 0.0 with dgcpann

### ISSUE THREAD

**Issue by ragnorc** [darioizzo/dcgp#86]
**Describe the bug**
For some reason, loss(X,y, "CE") always returns 0.0 but works fine for MSE.

**To Reproduce**
1. Create a network using dcgpy.expression_ann with 2 inputs and 1 output
2. Run the loss function with any valid input. E.g.: loss([[0.5, 0.6]], [[1]], "CE")

**Expected behavior**
A non-zero cross-entropy loss.

**Environment (please complete the following information):**
 - Installation method: [conda]
 - Version: [latest]

**Comment by darioizzo** [comment]
The cross-entrpy is a loss used to classify. When the network only has one output this correspond is a one class classification problem. Whatever outputs of the network its always correct as the class is only one!! SO this is not a bug but intended behaviour.
